lib/tasks/schedule.rake

```diff
@@ -111,4 +111,11 @@ namespace :schedule do
     next unless Date.today.sunday?
     Rake::Task['sitemap:refresh'].invoke
   end
+
+  desc 'fetch and assign labels for repos'
+  task fetch_labels_and_assign: :environment do
+    Repo.find_each(batch_size: 100) do |repo|
+      RepoLabelAssigner.new(repo: repo).create_and_associate_labels!
```

Comment:
I considered creating a new background job here and enqueueing it for 10 repos at a time but worried about rate-limiting and not knowing the concurrency of background processing in production. I also wasn't sure what the total count of repos in production was.

Reply:
Repo count is on the main page it's ~6,000. The way we do API requests is that we rotate through everyone's tokens on the service for each request. https://github.com/codetriage/CodeTriage/blob/31bd0e7f109803b01d7dfcd553b59c08d8d9707c/config/initializers/git_hub_bub.rb#L48-L54 

It's a good idea not to make a bunch of extra requests, but we're not that constrained. If this is N API requests where N is the number of repos in the system, that's absolutely no problem.